_includes/reviews.html

```diff
@@ -1,9 +1,8 @@
-<h2>Reviews and Testimonials</h2>
-<ul> 
+<ul>
     {% for review in site.reviews %}
-    <li>
-        <h3>{{ review.name }}</h3>
-        <p>{{ review.text }}</p>
-    </li> 
+    <li class="hide-bullets">
+      <p class="review-paragraph">{{ review.text }}</p>
+        <h3 class="review-title">- {{ review.name }}</h3>
```

Comment:
This H3 should be at the same leve as the P tag. There's no nesting happening here.